packages/test-cases/cases/apis/wasm-rs/number-types/query/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+pub mod w3;
+pub use w3::*;
+
+pub fn i8_method(input: InputI8Method) -> i8 {
+    input.first + input.second
+}
+
+pub fn u8_method(input: InputU8Method) -> u8 {
+    input.first + input.second
+}
+
+pub fn i16_method(input: InputI16Method) -> i16 {
+    input.first + input.second
+}
+
+pub fn u16_method(input: InputU16Method) -> u16 {
+    input.first + input.second
+}
+
+pub fn i32_method(input: InputI32Method) -> i32 {
+    input.first + input.second
+}
+
+pub fn u32_method(input: InputU32Method) -> u32 {
```

Comment:
Some tests I noticed are missing are floats, 64 bit units and ints, String and Bytes

Reply:
We don't currently support floats & 64 bit integers. Also String + Bytes cannot have under/overflows in the same way integers can.